frontend/src/components/Header.js

```diff
@@ -1,45 +1,20 @@
 import React from 'react';
 
-import {
-  makeStyles,
-  Toolbar,
-  AppBar,
-  Typography,
-  InputBase,
-} from '@material-ui/core';
+import { makeStyles, Toolbar, AppBar } from '@material-ui/core';
 
 const useStyles = makeStyles((theme) => ({
   root: {
     flexGrow: 1,
     justifyContent: 'space-between',
   },
-  logo: {
-    fontWeight: 'bold',
-  },
-  search: {
-    backgroundColor: '#E5E5E5',
-    flexBasis: '25%',
-    paddingLeft: '10px',
-    paddingTop: '5px',
-    paddingBottom: '5px',
-  },
 }));
 
-export default function Header() {
+export default function Header({ children }) {
```

Comment:
Nice refactor, I've never seen this done before but it's great not to have to prop drill!